ceph_deploy/mds.py

```diff
@@ -122,6 +122,8 @@ def create_mds(distro, name, cluster, init):
     if distro.is_el:
         system.enable_service(distro.conn)
 
+    if distro.distro == 'suse':
+        system.enable_service(distro.conn, 'ceph')
```

Comment:
Personally I much prefer the change you propose.

Changing 

distro.is_el -> distro.is_rpm

I would like to see distro.is_el deprecated and removed if at all possible.